adserver/api/serializers.py

```diff
@@ -60,6 +60,10 @@ class AdDecisionSerializer(serializers.Serializer):
     # This purposefully doesn't use a URLField so we can disregard invalid values rather than rejecting them
     url = serializers.CharField(max_length=256, required=False)
 
+    # The placement index (0-indexed)
+    # 1 or more means there's multiple placements on this page
+    index = serializers.IntegerField(required=False, min_value=0, max_value=9)
```

Comment:
Can we not have a max here? It seems unlikely, but I know the PSF has a use case where they're showing more than 9 images in a page.

Reply:
Sure. I can remove it. I'd rather have a very large max (99 or even 999) than no max at all though if that makes sense though.